src/include/duckdb/common/types/data_chunk.hpp

```diff
@@ -49,6 +54,9 @@ class DataChunk {
 	//! The vectors owned by the DataChunk.
 	vector<Vector> data;
 
+	//! Options for sampling pushdown
+	SamplingPushdownOption sampling_pushdown_option;
```

Comment:
The `DataChunk` is the wrong location for this - can we move this somewhere else, e.g. to the `CollectionScanState`?